### ISSUE TITLE
Hide track play icon for anonymous users

### ISSUE THREAD

**Issue by eric-gilbertson** [RocketMan/zookeeper#175]
Hide track play icon for anonymous users. Since the URLs may point to resources that are restricted to station staff, eg those stored on GDrive, it makes sense to hide the play icon for anonymous users since they will not be able to access the track thus  causing user frustration. The icon space should be collapsed when they are not visible.

**Comment by RocketMan** [comment]
You know, I'm of two minds about this.  My initial inclination was that it should be restricted to logged-in users.  On the other hand, one could make the argument that it provides value to listeners, to be able to jump to artist and track resources directly from zk.  For a similar reason, it benefits artists that their material is cross-referenced and more accessible in this way.

Right now, is there any GDrive content that is not publicly accessible?  The LIDs etc that you linked to (thanks for that, btw!) seem to play for me without being logged in.  Do we anticipate adding any, and if so, maybe we can look at restricting this when that time comes?

It's easy enough to condition display of the functionality on isAuth('u').  Before we do that, though, I want to make sure we don't give up the advantages of having this feature visible to the public, if we don't have to.

**Comment by eric-gilbertson** [comment]
The genesis for this change was a concern from Francis that only station staff should be able to play spots. This coupled with the desire by Bill to someday store the A file in a permission restricted GDrive account means that at least some links will not work for anonymous users. A compromise would be to hide the play icon only for GDrive URLs in order to allow anonymous users the ability to play tracks from YouTube but I do wonder if there is much call for this.

**Comment by RocketMan** [comment]
I just pushed 1ab34ee3935 to address this for now.  The spots as well as anything else on google drive no longer shows for non-authenticated users.  As well, playable link promotion #172 is suppressed for all non-authenticated users.

**Comment by eric-gilbertson** [comment]
I had entered this ticket so that I would have something to check in both this change and the style fixes that we discussed via email. Thus I would appreciate it if you would refrain from making any additional changes in this page until my PR is accepted in order to save me another round of code merging & testing. 

Going forward I will assign such tickets to myself so that you will know that I will be handling the issue, eg as opposed to the search change ticket which I assigned to you since it seems like something that you would be best equipped to handle.
